src/main/java/ch/njol/skript/expressions/ExprScriptFileErrors.java

```diff
@@ -0,0 +1,67 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Last Reloaded Errors")
+@Description("The errors from the last singular or multiple scripts that were reloaded or enabled via an effect.")
+@Example("""
+	reload script named "test.sk"
+	if last reloaded errors is set:
+		disable script file "test.sk"
+	""")
+@Example("""
+	enable script named "-test.sk"
+	if last enabled errors is set:
+		disable script file "test.sk"
+	""")
+@Since("INSERT VERSION")
+public class ExprScriptFileErrors extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprScriptFileErrors.class, String.class, ExpressionType.SIMPLE,
+			"[the] last (:reloaded|enabled) [script|skript] errors");
```

Comment:
I don't think reloaded and enabled should be treated differently. Reloading is simply disabling and enabling. There's no actual difference between the two. Skript should not be an option. Script should probably be required. This could also be easily merged with ExprPaseError